k8s/workloads/discourse/discourse-ingress-prod.yaml

```diff
@@ -0,0 +1,67 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ingress-nginx
+  namespace: discourse-prod
+  labels:
+    app: discourse
+    env: prod
+spec:
+  releaseName: discourse-ingress-nginx
+  chart:
+    repository: https://kubernetes.github.io/ingress-nginx
+    name: ingress-nginx
+    version: "3.33.0"
+  values:
+    controller:
+      useIngressClassOnly: true
+      enableCustomResources: false
+      autoscaling:
+        enabled: true
+        minReplicas: 1
+        maxReplicas: 4
+        targetCPUUtilizationPercentage: 80
+        targetMemoryUtilizationPercentage: 80
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 200m
+          memory: 256Mi
+      admissionWebhooks:
+        enable: false
+      scope:
+        enabled: true
+      config:
+        enable-vts-status: true
+        force-ssl-redirect: true
+        # Rate limitting: defaults x7
+        limit-connections: "140"
+        limit-rpm: "1400"
+        limit-rps: "84"
+        proxy-body-size: 4m
+        proxy-buffer-size: 16k
+        # restrict this to the IP addresses of ELB
```

Comment:
I see the comment in many of the helm releases but the ip is `0.0.0.0/0` is this a TODO before we launch or anything? Curious where this comment comes from

Reply:
yeah, this is what we were messing with for the ingress-nginx helmrelease for moderator before.

the cluster should be internal traffic only, so this being 0.0.0.0/0 should not matter. If it does end up mattering, can update to use the VPC's ip cidr block (172.16.0.0/16)

Reply:
updated this to VPC internal cidr block